ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/EntityTypeToggleGroup.tsx

```diff
@@ -19,17 +20,23 @@ function EntityTabToggleGroup({
     cveCount = 0,
     imageCount = 0,
     deploymentCount = 0,
+    onChange,
 }: EntityTabToggleGroupProps) {
     const [activeEntityTabKey, setActiveEntityTabKey] = useURLStringUnion('entityTab', entityTabs);
 
+    function handleEntityTabChange(entityTab: EntityTab) {
```

Comment:
On the ImageCVE page, we need to reset the table sort and page number when changing between `Image` <-> `Deployment` tables. This callback allows us to make the change when the tab is clicked.